pkg/kube/watcher.go

```diff
@@ -46,8 +49,13 @@ const (
 // WatchServcies watches Kubernetes for NodePort and LoadBalancer services
 // and create listeners on 0.0.0.0 matching them.
 // Any connection errors are ignored and retried.
-func WatchForServices(ctx context.Context, tracker *tcplistener.ListenerTracker, configPath string,
-	portTracker *tracker.PortTracker, k8sServiceListenerIP net.IP,
+func WatchForServices(
+	ctx context.Context,
+	configPath string,
+	k8sServiceListenerIP net.IP,
+	enablePrivilegedService bool,
+	portTracker *tracker.PortTracker,
+	tracker *tcplistener.ListenerTracker,
```

Comment:
Please either rename `tracker` to `listenerTracker`, or rename `portTracker` to not contain `tracker`. I realize that existed before this PR, but it's just… kind of confusing to get into.